quax/examples/named/_core.py

```diff
@@ -4,6 +4,7 @@
 
 import equinox as eqx
 import jax.core
+import jax.extend
```

Comment:
Another nit, by convention this is `import jax.extend as jex`

Reply:
SGTM. I'll make a https://docs.astral.sh/ruff/settings/#lint_flake8-import-conventions_extend-aliases

Reply:
Done.